src/Components/Email/Image/Render.js

```diff
@@ -21,8 +56,8 @@ const ImagePreview = ({
       <img
         className={classes.root}
         src={src}
-        width={`${calculatedWidth}px`}
-        height={`${calculatedHeight}px`}
+        width={placeholderDimension(src, width, calculatedWidth)}
```

Comment:
Do we need to handle placeholder src here? I'm not sure if when in "email mode" if I'm rendering an Email/Image from the frontend or if I'm loading a normal Image component. If I am rendering Email/Image, it seems like we would need to handle replacing the src with placeholder here if we receive a string replacement tag that isn't converted yet (in edit mode)

Reply:
so this is preview only not edit mode. The base image component is used in editor mode.